onnxruntime/core/flatbuffers/schema/ort.fbs

```diff
@@ -202,10 +207,11 @@ table Model {
   domain:string;
   model_version:int64;
   doc_string:string;
-  
+
   graph:Graph;
-  
+
   graph_doc_string:string;
+  metadata_props:[StringStringEntry];
 }
```

Comment:
Can you regenerate the python bindings as well? 

In theory, you can just run \onnxruntime\core\flatbuffers\create_python_bindings.py to handle that.